Pull request: LTI provider

luukverhoeven opened the pull request:
Added LTI grade reset support. More information on #30

danmarsden commented:
Hi @luukverhoeven - sorry for the time it's taken to look at this! - seems to be a massive amount of stuff in my worklog at the moment! :-) 

It looks like this patch is relying on the archivecompletiondata setting which is supposed to be just for the course_completions and course_completion_crit_compl tables. 
Each Plugin that doesn't have it's own archive has it's own "Archive old attempts" checkbox that allows the teacher to decide if we need to archive those activity specific records.

I see a couple of coding guideline violations in travis checks too.

Let me know if you're not able to find the time to fix those things up and I'll try to find some time to come back to this again at some point (likely to be next year)

thanks!

luukverhoeven commented:
Hi @danmarsden, 

Thanks for checking the changes in busy times. The **archivecompletiondata** shouldn't be used, I see this now. I removed the check on it. Old LTI grades will now always archived if **Reset LTI is enabled**

Also the Travis validation was something I have overlooked. I have updated the code.

Thank you.

Greets Luuk
